app/representers/member_permissions_representer.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+# This representer represents member permissions in a user-friendly way
+# Examples:
```

Comment:
I am not sure if this comment here is a good idea 🤔 It is too easy to get it outdated and it is not difficult to see this data is anybody wants to see it (specially from the tests).
However, either with this comment or without it, I think it would be good to add this info to the description of the PR or a comment in the PR itself 😄

Reply:
Well, I think that it's not clear at all from the code how the resulting output would look like. For example, you would need to dive into details of how `collection` work and what `as:` and `wrap:` mean in its context...

Reconstructing from tests would also be time consuming. I think a clear example wouldn't hurt. And it's "close" to the code, so I think if somebody changes the API, he can easily add a line or two (also it would be a good visual diff to see what exactly changed). As we don't have documentation of the responses in the ActiveDocs (sadly)...

I agree that it'd be good to add it to the PR description 👍 Will do.